test/test-fs-copyfile.c

```diff
@@ -184,11 +184,14 @@ TEST_IMPL(fs_copyfile) {
   ASSERT(r == 0);
   handle_result(&req);
 
+  /* Don't test on Windows until we got ReFS set up. */
+#if !defined(_WIN32) || defined(UV_WIN32_WE_CAN_HAZ_REFS)
```

Comment:
Can you remove this? We're already testing this, and should continue to do so.

Reply:
Sure.